### PR TITLE
Fix/blocked user

### PR DESCRIPTION
**Pull request by Adamik10**
#### Link to issue
https://reload.atlassian.net/browse/DDFSOEG-401

#### Description
This PR adds functionality to show disabled reservation button on the material page to blocked users. 

#### Screenshot of the result
image

#### Checklist
- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions
n/a

### CODE REVIEW

**Review by kasperg — CHANGES_REQUESTED** [review]
:+1: Nice with the added test but please reconsider the checking logic.

File: src/components/material/material-buttons/physical/MaterialButtonsPhysical.tsx [hunk 1]
```diff
@@ -24,37 +28,47 @@ const MaterialButtonsPhysical: React.FC<MaterialButtonsPhysicalProps> = ({
...
+        size={size}
+      />
+    );
+  }
+
+  if (!userData) {
+    return null;
+  }
+
+  return <MaterialButtonUserBlocked size={size} dataCy={dataCy} />;
```
[the remark this answers is not in the document]
Comment (Adamik10) [reply src/components/material/material-buttons/physical/MaterialButtonsPhysical.tsx:71]: I can't believe I was overcomplicating it for myself so much.

File: src/core/utils/types/error.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export type ErrorFbs = {
+  message: "string";
+};
```
Comment (kasperg) [inline src/core/utils/types/error.ts:3]: Please remove this type. It is no longer used.

**Review by kasperg — APPROVED** [review]
:+1: but please remember to remove the unneeded type before you pass this along.

**Review by jeppekroghitk — APPROVED** [review]
LGTM :)

**Pull request merged**